frontend/src/bundles/common/components/radio/radio.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+} from 'react-hook-form';
+
+import { useFormController } from '../../hooks/hooks.js';
+
+type Properties<T extends FieldValues> = {
+    name: FieldPath<T>;
+    id: string;
+    label: string;
+    value: string;
+    control: Control<T>;
+};
+
+const Radio = <T extends FieldValues>({
+    name,
+    id,
+    label,
+    value,
+    control,
+}: Properties<T>): JSX.Element => {
+    const { field } = useFormController({ name, control });
+
+    return (
+        <div className="flex items-center py-[3px]">
```

Comment:
use relative sizes as `rem`

Reply:
changed padding to tailwindcss size 1

But I don't understand where error state message should be? You mean error state when radio will be red like in input? If yes, then I should implement all states like in input?
![2024-02-13_10-19](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/26843161/b04208e5-923d-4714-81e9-f3d3a01d8e71)